autoload/ale/linter.vim

```diff
@@ -25,6 +25,7 @@ let s:default_ale_linters = {
 \   'csh': ['shell'],
 \   'go': ['gofmt', 'golint', 'go vet'],
 \   'help': [],
+\   'llvm': ['llc'],
```

Comment:
You can remove this line. Without any entries in the Dictionary here, ALE will run all available tools for linting. The only reason to specify something here is to _disable_ some linter by default.

Reply:
I'll remove the line, thanks.